Firestore/Source/Model/FSTMutation.mm

```diff
@@ -142,7 +140,7 @@ - (BOOL)isEqual:(id)other {
 }
 
 - (NSUInteger)hash {
-  NSUInteger result = [self.key hash];
+  NSUInteger result = self.key.Hash();
```

Comment:
```c++

using firebase::firestore::util::Hash;

- (NSUInteger)hash {
  return Hash(self.key, self.precondition, [self.value hash]);
}
```

Reply:
Done. I left the transform one because it iterates an array.